requirements.txt

```diff
@@ -1,7 +1,7 @@
-cachetools==5.0.0
-jinja2>=3.0.0
-netaddr>=0.8.0
+cachetools~=5.0
+jinja2~=3.0
+netaddr~=0.8
 pyjwt[crypto]>=1.6.0
```

Comment:
Latest version is `2.9.0`. Maybe our code actually relies on some 2.x API and that's why you were seeing issues when pinning with `~=`. Should check how it's used in the code and compare to the pyjwt release notes. Links can be found here:

https://pypi.org/project/PyJWT/

Reply:
Tested with pyjwt version `2.9.0` and it worked so I set it to `~=2.0`